magda-web-client/src/_variables.scss

```diff
@@ -9,7 +9,7 @@ $magda-color-dark: darken($magda-color-primary, 15%);
 $AU-color-foreground-action: $magda-color-light;
 $AU-color-foreground-focus: $magda-color-dark;
 $AU-color-foreground-text: $magda-color-dark;
-$AU-color-background: #ffffff;
```

Comment:
If we change here, probably gonna impact a lot of places. 
e.g. search box, search result page etc.
![image](https://user-images.githubusercontent.com/674387/66543262-a1899380-eb80-11e9-9cca-d3c7897a2193.png)
I think we only want to update the background for the loading area? @philipgrimmett  please comment on this (here is a test site: https://1596-map-preview-background.dev.magda.io/)

Reply:
@t83714 yep, my intention was to only apply the grey background to loading components like the embedded map. I'd suggest doing the same for other components you expect might take some time to load (in the absence of actual skeleton screens). Ping @alyce

Reply:
@t83714 Yep, that's fair. I have added a new color instead for preview components.